app/Http/Controllers/Admin/MeetingCrudController.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Requests\MeetingRequest;
+use App\Models\Meeting;
+use Backpack\CRUD\app\Http\Controllers\CrudController;
+use Backpack\CRUD\app\Library\CrudPanel\CrudPanelFacade as CRUD;
+use Backpack\CRUD\app\Library\Widget;
+use Carbon\Carbon;
+
+/**
+ * Class MeetingCrudController.
+ *
+ * @property-read \Backpack\CRUD\app\Library\CrudPanel\CrudPanel $crud
+ */
+class MeetingCrudController extends CrudController
+{
+    use \Backpack\CRUD\app\Http\Controllers\Operations\ListOperation;
+    use \Backpack\CRUD\app\Http\Controllers\Operations\CreateOperation;
+    use \Backpack\CRUD\app\Http\Controllers\Operations\UpdateOperation;
+    use \Backpack\CRUD\app\Http\Controllers\Operations\DeleteOperation;
+    use \Backpack\CRUD\app\Http\Controllers\Operations\ShowOperation;
+    use \Backpack\CalendarOperation\CalendarOperation;
+
+    use \App\Http\Controllers\Admin\Operations\SMSOperation;
+
+    /**
+     * Configure the CrudPanel object. Apply settings to all operations.
+     */
+    public function setup(): void
+    {
+        CRUD::setModel(Meeting::class);
+        CRUD::setRoute(config('backpack.base.route_prefix').'/meeting');
+        CRUD::setEntityNameStrings(__('meeting'), __('meetings'));
+    }
+
+    public function getMeetingFieldsMap()
+    {
+        return [
+            'title'            => 'title',
+            'start'            => 'start',
+            'end'              => 'end',
+            'background_color' => 'background_color',
+            'text_color'       => 'text_color',
+            'all_day'          => 'all_day',
+            'number'           => 'number',
+        ];
+    }
+
+    public function setupMeetingOperation()
+    {
+        $this->crud->setOperationSetting('initial-view', 'dayGridMonth');
+
+        $this->crud->setOperationSetting('views', ['dayGridMonth', 'timeGridWeek', 'timeGridDay']);
+
+        $this->crud->setOperationSetting('editable', true);
+
+        $this->crud->setOperationSetting('background_color', '#3788d8');
+
+        $this->crud->setOperationSetting('text_color', '#ffffff');
+
+        $this->addMeetingLineButton(
```

Comment:
I think you meant to call `addCalendarLineButton()` ?

Reply:
Yes I did 😅
Fixed!